Makefile

```diff
@@ -77,12 +77,12 @@ ifeq ($(SGX_PRODUCTION), 1)
 	SGX_ENCLAVE_MODE = "Production Mode"
 	SGX_ENCLAVE_CONFIG = "enclave-runtime/Enclave.config.production.xml"
 	SGX_SIGN_KEY = $(SGX_COMMERCIAL_KEY)
-	WORKER_FEATURES = --features=production,$(WORKER_MODE),$(ADDITIONAL_FEATURES)
+	WORKER_FEATURES += --features=production,$(WORKER_MODE),$(ADDITIONAL_FEATURES)
```

Comment:
Can you explain this change? Is his for something in you have in the mind for the future?

Reply:
Yeah, basically, if somebody wants to have the `dcap` features activated they would have to use the following command:
```bash
SGX_MODE=HW WORKER_FEATURES=dcap make
```
The `dcap` value would be lost/overwritten as of now.

But the `+=` trick is not the right one either as it will add a space as well, which will break things... I will submit a proper patch for it soon.

Reply:
Thanks for the explanation, in this case I will merge #1196 first, and we can patch this PR soon.

Reply:
Solved it in 333d4e7.